Test/git-issues/git-issue-2612B.dfy

```diff
@@ -0,0 +1,8 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
```

Comment:
Can you make the `B` at the end of this filename lowercase? That's the style we've used before.

Reply:
Done